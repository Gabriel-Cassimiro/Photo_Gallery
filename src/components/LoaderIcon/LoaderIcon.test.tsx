/**
 * @jest-environment jsdom
 */

import { render } from "@testing-library/react"
import { LoaderIcon } from "./index"

describe("Test Loading component", () => {
	it("Should render correctly", () => {
		const { container } = render(<LoaderIcon />)
		expect(container).toMatchSnapshot()
	})
})
