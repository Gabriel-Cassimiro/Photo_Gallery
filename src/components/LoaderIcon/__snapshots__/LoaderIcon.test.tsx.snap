// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Loading component Should render correctly 1`] = `
<div>
  <div
    class="flex justify-center items-center"
  >
    <div
      class="animate-spin rounded-full h-32 w-32 border-b-2 border-gray-900"
    />
  </div>
</div>
`;
